GDM Bootcamp RPG/Assets/Scripts/PlayerController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[RequireComponent(typeof(PlayerMotor))]
+public class PlayerController : MonoBehaviour
+{
+    public LayerMask movementMask;
+
+    Camera cam;
+    PlayerMotor motor;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        cam = Camera.main;
+        motor = GetComponent<PlayerMotor>();
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+
+        if (Input.GetMouseButtonDown(0))
+        {
+            Ray ray = cam.ScreenPointToRay(Input.mousePosition);
+            RaycastHit hit;
+
+            // TODO: May need to explain what the "out" keyword does. Passing by value / ref
+            if (Physics.Raycast(ray, out hit, 100, movementMask))
+            {
+                motor.MoveToPoint(hit.point);

```

Comment:
Include WASD controls
Do we want to show rigidbody w/ AddForce or velocity real fast?